docs/commands/cookie-cut.asciidoc

```diff
@@ -33,7 +33,7 @@ cp $HOOT_HOME/test-files/DcTigerRoads.osm DcSmall.osm
 # It is a good idea to manually verify the alpha shape for correctness after it
 # is generated. -15m works well for this square data, but will likely cause
 # issues on more complex shapes.
-hoot generate-alpha-shape DcSmall.osm 1000 -15 DcSmallBounds.shp
+hoot generate-alpha-shape --alpha 1000 --buffer -15 DcSmall.osm DcSmallBounds.shp
```

Comment:
Updated the command line arguments to make a little more sense using `--alpha` followed by the alpha value and same with the `--buffer` value.  Then we don't just have a couple of numbers hanging out there without explanation.